site/docs/tools/spark_auth_config.md

```diff
@@ -0,0 +1,55 @@
+# Authentication in Spark
+
+When Nessie is used in Spark-based environments (either with [Iceberg](./iceberg/index.md) 
+or [Delta Lake](./deltalake/index.md)) the Nessie authentication settings are configured via Spark session properties.
+
+=== "Java"
+    ``` java
+    // local spark instance, assuming NONE authentication
+    conf.set("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog")
+        .set("spark.sql.catalog.nessie.authentication.type", "NONE")
+        .set(...);
+    spark = SparkSession.builder()
+                        .master("local[2]")
+                        .config(conf)
+                        .getOrCreate();
+    ```
+=== "Python"
+    ``` python
+    # local spark instance, assuming NONE authentication
+    spark = SparkSession.builder \
+            .config("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog") \
+            .config("spark.sql.catalog.nessie.authentication.type", "NONE") \
+            .config(...) 
+            .getOrCreate()
+    ```
+
+The `spark.sql.catalog.nessie` prefix identifies properties for the Nessie catalog. The `nessie` part is just
+the name of the catalog in this case (not to be confused with the Nessie project name).
+
+# Authentication Settings
+
+The value of the `authentication.type` property (with the appropriate prefix) can be one of the following:
+
+* NONE
+* BEARER
+* AWS
+
+## Authentication Type "NONE"
+
+For the Authentication Type "NONE" only the `authentication.type` property needs to be set
+(with the appropriate prefix).
```

Comment:
same for complete example incl a prefix

Reply:
Prefixes are discussed at the top, in the section about Spark sesssion settings